### ISSUE TITLE
The loss cost is negative

### ISSUE THREAD

**Issue by haitaoWi** [cvg/diffmvs#11]
Hello, I would like to ask how to handle the situation where the loss cost is negative when training casdiffmvs on my own private dataset? The final depth error was very high.

**Comment by FangjinhuaWang** [comment]
Hi,

The loss could be negative because we have log loss on confidence in our[ loss function](https://github.com/cvg/diffmvs/blob/cd10d5c282a9cabd45a2f64598cd2b990b408d35/models/loss.py#L65).
